src/views/Instance/styleCytoViz.js

```diff
@@ -43,3 +43,14 @@ export const getDefaultSelectedNodeStyle = (theme) => ({
   height: NODE_SELECTED_ICON_SIZE,
   'background-blacken': -NODE_SELECTED_BLACKEN_RATIO,
 });
+
+export const getDefaultCompoundNodeStyle = (theme) => ({
+  'border-style': 'dashed',
+  'border-color': EDGE_DEFAULT_COLOR,
+  'border-width': 3,
+});
+export const getDefaultCollapsedCompoundNodeStyle = (theme) => ({
+  'border-style': 'solid',
+  'border-color': EDGE_DEFAULT_COLOR,
```

Comment:
We should probably define a new constant `COMPOUND_DEFAULT_COLOR` instead of using `EDGE_DEFAULT_COLOR` here :smile: